internal/app/singularity/remote_get_login_password.go

```diff
@@ -0,0 +1,59 @@
+package singularity
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	scslibclient "github.com/sylabs/scs-library-client/client"
+	"github.com/sylabs/singularity/pkg/sylog"
+)
+
+var errNotSupported = errors.New("This library does not support direct OCI access")
+
+// RemoteGetLoginPassword retrieves cli token from oci library shim
+func RemoteGetLoginPassword(config *scslibclient.Config) (string, error) {
+	client := http.DefaultClient
+	path := "/v1/rbac/users/current"
+	endPoint := config.BaseURL + path
+
+	req, err := http.NewRequest(http.MethodGet, endPoint, nil)
+	if err != nil {
+		return "", errNotSupported
+	}
```

Comment:
This sends a common error message to the user, but we don't have any logging or handling of any of the errors, so theres no way to find out why the request fails.